docker/latest/docker-compose-ipld-eth-server.yml

```diff
@@ -2,22 +2,26 @@ version: "3.2"
 services:
   ipld-eth-server:
     restart: unless-stopped
+    depends_on:
+      - ipld-eth-db
     image: vulcanize/ipld-eth-server:v2.0.0
```

Comment:
Can update version to [v3.2.2](https://hub.docker.com/layers/ipld-eth-server/vulcanize/ipld-eth-server/v3.2.2/images/sha256-bacb17c6654b0f3587e7eef42c052aa0311304e8919cd57e2f23458fb7c709d3?context=explore)